bridge/qb/server/main.lua

```diff
@@ -85,15 +85,31 @@ function qbCoreCompat.Functions.TriggerCallback(name, source, cb, ...)
     qbCoreCompat.ServerCallbacks[name](source, cb, ...)
 end
 
----@deprecated call server function SpawnVehicle instead from imports/utils.lua.
+---@deprecated call server function qbx.spawnVehicle from modules/lib.lua
 qbCoreCompat.Functions.CreateCallback('QBCore:Server:SpawnVehicle', function(source, cb, model, coords, warp)
-    local netId = SpawnVehicle(source, model, coords, warp)
+    model = type(model) == 'string' and joaat(model) or (model --[[@as integer]])
+    local ped = GetPlayerPed(source)
+
+    local netId = qbx.spawnVehicle({
+        model = model,
+        spawnSource = coords or ped,
+        warp = warp and ped or nil,
+    })
+
     if netId then cb(netId) end
 end)
 
----@deprecated call server function SpawnVehicle instead from imports/utils.lua.
+---@deprecated call server function qbx.spawnVehicle from modules/lib.lua
 qbCoreCompat.Functions.CreateCallback('QBCore:Server:CreateVehicle', function(source, cb, model, coords, warp)
-    local netId = SpawnVehicle(source, model, coords, warp)
+    model = type(model) == 'string' and joaat(model) or (model --[[@as integer]])
+    local ped = GetPlayerPed(source)
+
+    local netId = qbx.spawnVehicle({
+        model = model,
+        spawnSource = coords or ped,
+        warp = warp and ped or nil,
+    })
```

Comment:
```suggestion
    local netId = qbCoreCompat.Functions.CreateVehicle(source, model, nil, coords, warp)
```
Same here but a kinda different function.